lib/src/ruby/module.cc

```diff
@@ -113,7 +116,7 @@ extern "C" {
         set_level(log_level::warning);
 
         // Initialize ruby
-        auto ruby = facter::ruby::api::instance();
+        auto ruby = api::instance();
```

Comment:
We need to set `api::ruby_lib_location` at line 138 below in `module::module`, based on FACTER_RUBY if defined.

It's not needed here because Init_libfacter is run from inside Ruby, but this is the closest spot to line 138.